docs/guides/admin/docs/workflowoperationhandlers/duplicate-event-woh.md

```diff
@@ -0,0 +1,48 @@
+Duplicate Event Workflow Operation
+===============================
+
+```Id: duplicate-event```
+
+Description
+-----------
+
+The duplicate event operation can be used to duplicate an event by copying an existing one. The main use case are events,
+which contain a series of different presentations which were all recorded with just one recording. In order to create
+seperate events for each presentation, the original recording can be copied and each copy can be cut to only contain
+one presentation. This operation only works with events, which were not published, yet.
```

Comment:
Why should this only work with events that are not published yet?

Please add some text that describes the handling of existing publications.